package-lock.json

```diff
@@ -1,8 +1,15495 @@
 {
     "name": "alexa-skills-kit-toolkit",
     "version": "2.7.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
Can we separate concerns here? Let's not conflate a change to the lock file format with releasing APL 1.7.

Reply:
The package lock file is auto generated. Should I not including the package.lock file in the PR?

Reply:
The package lock is because you're using `npm 7`. Try re-installing with `npm 6` for this PR